pkg/compiler/beancount/compiler.go

```diff
@@ -111,10 +111,15 @@ func (b *BeanCount) writeHeader(file *os.File) error {
 	}
 
 	accounts := b.GetAllCandidateAccounts(b.Config)
-	for k := range accounts {
-		if k == "" {
-			continue
-		}
+    var sortedAccounts []string
+    for k := range accounts {
+        if k != "" {
+            sortedAccounts = append(sortedAccounts, k)
+        }
+    }
+    sort.Strings(sortedAccounts)
+
+	for _, k := range sortedAccounts {
```

Comment:
The indent for line 114-120 and 122 are different. Can u change the indent to which file original used(I think is tab)?